packages/nimbus-common/constants.js

```diff
@@ -6,7 +6,7 @@ exports.EXTS_GROUP = '{ts,tsx,js,jsx}';
 
 exports.TJSX_EXT_PATTERN = /\.m?(t|j)sx?$/;
 
-exports.ASSET_EXT_PATTERN = /\.(ttf|eot|otf|svg|woff|woff2|mp3|png|jpg|jpeg|gif|ico|css|scss)$/;
+exports.ASSET_EXT_PATTERN = /\.(ttf|eot|otf|svg|woff|woff2|mp3|png|jpg|jpeg|gif|ico)$/;
```

Comment:
Allows consumers to add their own css loaders/plugins.

Reply:
nice, I think this will be good based on usage so far 👍